<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_RetargetingBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Retargeting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Marker List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ui_markerListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="ui_MarkerSetBtn">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_MarkerResetBtn">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ctrl List    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ui_ctrlListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="ui_CtrlSetBtn">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_CtrlResetBtn">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Start Frame:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_StartFrameSpinBox">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ui_ForeHeadEndFrameLabel">
         <property name="text">
          <string>End Frame:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_EndFrameSpinBox">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>640</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ui_ReadyForTrainingBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Ready To Train</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ui_TrainingNoticeLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>* First training example shoud be neutral expression</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="ui_TrainingTitleLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Training</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="ui_TrainingSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ui_TrainingSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ui_TrainingTotalExampleNumberLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>/0</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="ui_TrainingAddBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_TrainingRemoveBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_TrainingSetNeutralBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Neutral</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ui_TrainingSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ui_TrainingSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_TrainingSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ui_TrainingSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
